### ISSUE TITLE
Feature request: Enable multiple strings to exclude items from the sorting process

### ISSUE THREAD

**Issue by TheoRetisch1** [MLG-Fortress/AutomaticInventory#37]
It would be great if in the config option "excludeItemsContainingThisString" one could enter not only one string but several strings. I have several plugins whose items should not be sorted away and it would be cool if one could exclude the corresponding items from the sorting process by name.

**Comment by RoboMWM** [comment]
Hmm, I didn't make it regex did I?

I should probably do that then.

**Comment by RoboMWM** [comment]
Sorry about this, I was waiting on the EHM guy for a response on his stuff since he asked earlier and thus had this lower priority - but is having it as a list (what the guy PR'd) instead of using regex acceptable? I'd assume it is since it's what you asked for.

**Comment by TheoRetisch1** [comment]
Yes, definitely, I know kikel. I'm excited about the feature 💯
